Financials/_posts/Quarterly-Report/2021-Q2/2021-09-11-2021-Q2.md

```diff
@@ -4,389 +4,230 @@ title:  "Yearn Finance 2021 Q2 Quarterly Report"
+- En mayo de 2021, Yearn incurrió en un gasto único de $66k en concepto de intereses para el protocolo Keep3r. Se trata de un préstamo que Keep3r concedió a Yearn en relación con el fondo de liquidez KP3R/ETH que Yearn utiliza para pagar a los Keepers que cosechan yVaults de Yearn. No esperamos que este gasto por intereses ocurra en el futuro.
+
+# El equipo de colaboradores a tiempo completo se amplió en el segundo trimestre con más desarrolladores, personal de marketing y operaciones.
 
 ![](page10.png)
 
-- The full-time contributor team has expanded and now consists of twenty-six individuals.
-- There are now thirteen developers, ten operations personnel, and one personnel each
-  for marketing, design, and legal.
-- Administrative salaries for Q2-21 was less than 5% of gross profit generated for the
-  quarter.
+- El equipo de colaboradores a tiempo completo se ha ampliado y ahora consta de veintiséis personas.
+- Ahora hay trece desarrolladores, diez empleados de operaciones y uno de marketing, diseño y legal.
+- Los salarios administrativos para el segundo trimestre del 21 fueron menos del 5% de la utilidad bruta generada para el trimestre.
 
-# Coordinape distributes the majority of Yearn community grants and is a novel and innovative approach to decentralized payments.
+# Coordinape distribuye la mayoría de las subvenciones comunitarias de Yearn y es un enfoque novedoso e innovador para los pagos descentralizados.
 
 ![](page11.png)
 
-- The majority of Yearn community grants are distributed by the Coordinapeprogram.
-  Coordinapeis a decentralized grant selection and disbursement model. Previously
-  awarded grant recipients and full-time contributors are each allocated 100 GIVE tokens.
-  These tokens can be allocated to contributors in the Yearn ecosystem. Salaried
-  contributors are ineligible to receive grants. The Coordinapebudget, approved by the
-  yBudgetteam, increased in Q2 and as of June 2021 is approximately $58k per month.
-- Redphoneassisted with communication writeups particularly on Yearn’stwitter.
-- There are consistently over 50 grant recipients each month from the Coordinape
-  program. Due to the large number of recipients we have presented the total Coordinape
-  grant figure instead each individual payment in the table at left.
-- You can access Coordinapehere: https://coordinape.com/
+-	La mayoría de las subvenciones comunitarias de Yearn son distribuidas por el programa Coordinape. Coordinape es un modelo descentralizado de selección y desembolso de subvenciones. A cada uno de los beneficiarios de subvenciones anteriores y a los contribuyentes a tiempo completo se les asignan 100 tokens GIVE. Estos tokens se pueden asignar a contribuyentes en el ecosistema Yearn. Los contribuyentes asalariados no son elegibles para recibir subvenciones. El presupuesto de Coordinape, aprobado por el equipo yBudget, aumentó en el segundo trimestre y en junio de 2021 es de aproximadamente $58k por mes.
```

Comment:
Remove the excesive space before "La mayoría de las subvenciones comunitarias..."